.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1216,6 +1218,7 @@ tsh bench web sessions --max=5000 --web user ls
 - Non-AD setup
   - [ ] Installer in GUI mode finishes successfully on instance that is not part of domain
   - [ ] Installer works correctly invoked from command line
+  - [ ] Non-AD instance can be added to `static_hosts` section in config file and is visible in UI
```

Comment:
Perhaps we should move this under `- Direct mode (set listen_addr):` above and have both

- [ ] Can connect to AD desktop defined in static `static_hosts` section.
- [ ] Can connect to Non-AD desktop defined in static `static_hosts` section.

Reply:
Yep, makes sense, I've changed it